ksqldb-parser/src/main/antlr4/io/confluent/ksql/parser/SqlBase.g4

```diff
@@ -297,14 +297,20 @@ primaryExpression
     | MAP '(' (expression ASSIGN expression (',' expression ASSIGN expression)*)? ')'     #mapConstructor
     | STRUCT '(' (identifier ASSIGN expression (',' identifier ASSIGN expression)*)? ')'  #structConstructor
     | identifier '(' ASTERISK ')'                              		                        #functionCall
-    | identifier '(' (expression (',' expression)* (',' lambdaFunction)*)? ')'            #functionCall
+    | identifier'(' expression ',' intervalExpression ')' 						                    #functionCall
+    | identifier '(' (expression (',' expression)*
+                     ((',' lambdaFunction)* | (',' intervalExpression)*))? ')'            #functionCall
     | value=primaryExpression '[' index=valueExpression ']'                               #subscript
     | identifier                                                                          #columnReference
     | identifier '.' identifier                                                           #qualifiedColumnReference
     | base=primaryExpression STRUCT_FIELD_REF fieldName=identifier                        #dereference
     | '(' expression ')'                                                                  #parenthesizedExpression
     ;
 
+intervalExpression
+    : valueExpression windowUnit
```

Comment:
I'm not sure about using `valueExpression` for intervals. We should support only numbers here, right?

Reply:
The reason was to support expressions like `4 + 4 DAYS` or `COL0 MINUTES`